src/index.html

```diff
@@ -11,12 +11,98 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="shortcut icon"
+      href="./images/logo.png"
+      type="image/x-icon"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="header__products">
+        <ul class="nav__links">
+          <li class="nav__link">
```

Comment:
you're using nav__links element without first defining nav block ([more info on how to use BEM](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en))